### PR TITLE
Adding Quick Setup option

### PR DESCRIPTION
**Pull request by KendallCole**
Option to programmatically create the nessasary folders instead of having to manually add them

### CODE REVIEW

File: src/Utility/QuickSetup.lua
Comment (1Axen) [inline src/Utility/QuickSetup.lua]: Characters folder should use the name specified in settings (`CharacterFolder`).

File: src/Utility/QuickSetup.lua
Comment (1Axen) [inline src/Utility/QuickSetup.lua]: Same thing here.

File: src/Utility/QuickSetup.lua
Comment (1Axen) [inline src/Utility/QuickSetup.lua]: Small nit, would prefer dot index instead of colon here, it's just more inline with the rest of the code.

File: src/init.lua
Comment (1Axen) [inline src/init.lua]: Require should be moved out of here and into the imports section.

**Review by 1Axen — CHANGES_REQUESTED** [review]
I've left some comments on things that need changing before merging.

**Comment by KendallCole** [comment]
Made the requested changes <3

File: src/init.lua
Comment (1Axen) [inline src/init.lua]: I only meant for the `QuickSetupUtility` to be moved, moving the rest will cause them to error as they expect the characters and or visuals folder to exist which someone using the quick setup option might not have statically in their workspace.

**Review by 1Axen — CHANGES_REQUESTED** [review]
Breaking change needs resolving otherwise LGTM

**Comment by KendallCole** [comment]
Made the change :)

**Review by 1Axen — APPROVED** [review]

**Comment by 1Axen** [comment]
LGTM

**Pull request merged**
